components/collection/drop/GenerativeLayout.vue

```diff
@@ -74,6 +76,8 @@ import {
 } from '@/components/drops/useDrops'
 import { useCollectionActivity } from '@/composables/collectionActivity/useCollectionActivity'
 import { useCollectionMinimal } from '@/components/collection/utils/useCollectionDetails'
+import PartyModal from '@/components/common/party/PartyModal.vue'
```

Comment:
```suggestion

```

can be auto imported. 

![CleanShot 2024-04-08 at 10 00 05@2x](https://github.com/kodadot/nft-gallery/assets/44554284/a69edb9c-3ac9-41de-a0ec-4812868865af)